www/templates/control/main-control.html

```diff
@@ -140,23 +153,31 @@
         <div class="control-list-text" ng-i18next>{{'control.check-sensed-data'}}</div>
         <div ng-click="showSensed()" class="gray-icon control-icon-button"><i class="ion-ios-arrow-right"></i></div>
       </div>
+      <!-- replacing collection and sync with React DataTable-->
+      <!-- collection li and data table -->
       <div class="control-list-item">
         <div class="control-list-text" ng-i18next>{{'control.collection'}}</div>
         <div ng-click="editCollectionConfig($event)" class="gray-icon control-icon-button"><i class="ion-edit"></i></div>
       </div>
-      <ion-list>
-
+      
+      <!-- not sure what if any class(es) are appropriate here, chose these based off existing code, no particular change in appearance -->
+      <control-data-table control-data="settings.collect.show_config"></control-data-table>
+      
+      <!-- <ion-list>
           <ion-item class="row" ng-repeat="entry in settings.collect.show_config">
                 <div class="col timestamp item-text-wrap control-info">{{entry.key}}</div>
                 <div class="col detail item-text-wrap control-info"> {{entry.val}}</div>
           </ion-item>
-      </ion-list>
-
+      </ion-list> -->
+      
+      <!-- sync li and data table -->
       <div class="control-list-item">
         <div class="control-list-text" ng-i18next>{{'control.sync'}}</div>
         <div ng-click="editSyncConfig($event)" class="gray-icon control-icon-button"><i class="ion-edit"></i></div>
       </div>
-      <ion-list>
+      <control-data-table control-data="settings.sync.show_config"></control-data-table>
+      
+      <!-- <ion-list>
```

Comment:
nit: we should remove commented out code before the final commit 🔥, to avoid confusion down the road